platform/vs/docker-sonic-vs.dep

```diff
@@ -0,0 +1,10 @@
+#DPKG FRK
```

Comment:
We already have *.mk files to describe the dependence, could you reuse instead of creating new infrastructure?

Reply:
Agreed. We will address this in next phase. 
We wanted that the caching framework should be a plugin framework, so this can be added/removed dynamically. 
Also it will be easy to add more change into caching framework without disturbing the existing build framework.